routes/index.js

```diff
@@ -1,9 +1,58 @@
+require('dotenv').config();
 const apiRouter = require('express').Router();
+const jwt = require('jsonwebtoken');
+const {JWT_SECRET} = process.env;
+const {
+  getUserById
+} = require('../db');
+
+apiRouter.use(async (req, res, next) => {
+  const prefix = 'Bearer ';
+  const auth = req.header('Authorization');
+  if (!auth) {
+    next();
+  } else if (auth.startsWith(prefix)) {
```

Comment:
Add an `else` after this case, to take care of absolutely every situation.  Meaning like `else{ throw new Error('we could not log you in')}`

Reply:
Fixed